python/tk_multi_publish2/api/data.py

```diff
@@ -57,7 +57,7 @@ def to_dict(self):
 
         :return: A dictionary representing the data stored on the instance.
         """
-        return copy.deepcopy(self.__dict__)
+        return copy.copy(self.__dict__)
```

Comment:
deepcopy will crash on a template object. For now we're only going to do a shallow copy. to_dict is not public, so it isn't really that bad. it's only used during persistence.

Reply:
will this have side effects with other nested structures? what about UI? Custom data?

Reply:
This is only being used during persistence. Templates can't be deep-copied, which is why I'm dumbing this down to copy.